kaggle_environments/agent.py

```diff
@@ -134,15 +133,8 @@ def __init__(self, raw, environment):
         self.environment_name = environment.name
         self.raw = raw
         self.agent, self.is_parallelizable = build_agent(self.raw, self.builtin_agents, self.environment_name)
-        self.is_initialized = False
 
     def act(self, observation):
-        timeout = self.configuration.actTimeout
```

Comment:
We no longer need to keep track of initialization as used init time will automatically be deducted from overage.